dynamic_listings/python/default/containers/timer.py

```diff
@@ -3,13 +3,13 @@ class Timer:
     This is a simple timer class that executes a function after
     a certain amount of time
     """
-    time = 0
-    set_time = 0
+    time: float = 0
+    set_time: float = 0
     function_to_execute = None
     one_shot = False
     active = False
```

Comment:
Missing type hints, one should probably be `Callable[[], None]` (if I recall correctly) and two bool.